sdk/core/azure-core/src/main/java/com/azure/core/implementation/ResponseConstructorsCache.java

```diff
@@ -128,7 +131,8 @@
                                 bodyAsObject,
                                 null));
                         } catch (IllegalAccessException | InvocationTargetException | InstantiationException e) {
-                            throw logger.logExceptionAsError(Exceptions.propagate(e));
+                            throw logger.logExceptionAsError(new RuntimeException("Failed to deserialize 5-parameter"
+                                + "response without decoded headers. ", e));
```

Comment:
I noticed this because of the split, there isn't a space between "5-parameter" and "response". And same applies for all the other messages above. The split here doesn't hit the 120 character max either.